### DISCUSSION TITLE
Cannot log in via Facebook

### DISCUSSION THREAD

**Discussion by Sjsiisjsjskakkam** [thatskymod/Sky-CotL-Scripts#148] [General]
I can't log in to Facebook.And I can't access my email anymore, I want to log in via Facebook.

**Comment by thatskymod** [comment]
I hate to be that person, but let me suggest using a different login method. Google, Nintendo, and Local have proven to be reliable.

It's unclear why the Facebook option is still offered when it doesn’t function properly. I understand there may be challenges integrating Sky’s OAuth with a third-party package like Canvas, but that doesn’t justify listing it as an available option. You could raise this with the core [development team](https://github.com/artdeell/Canvas-Open-Source).

In the meantime, switching to another login provider should resolve the issue.
